### PR TITLE
New package: AJUpdateWatcher v. 2.0.4.0

### PR DESCRIPTION
**Pull request by tushev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57674)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/57668#pullrequestreview-941401399

File: manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: tushev.org.AJUpdateWatcher
+PackageName: AJUpdateWatcher
```
Comment (Trenly) [inline manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml:2]: This must match the value shown in control panel
```suggestion
PackageName: Update Watcher for AdoptOpenJDK
```

File: manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Moniker: aojdk-updatewatcher, ajupdatewatcher
+Author: Simon Tushev
+Publisher: tushev.org
```
Comment (Trenly) [inline manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml:5]: This must match the value shown in control panel
```suggestion
Publisher: Simon Tushev
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

File: manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: tushev.org.AJUpdateWatcher
+PackageName: AJUpdateWatcher
+Moniker: aojdk-updatewatcher, ajupdatewatcher
```
Comment (Trenly) [inline manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml:3]: Only one moniker is allowed
```suggestion
Moniker: aojdk-updatewatcher
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**